pkgs/tools/admin/azure-cli/default.nix

```diff
@@ -269,6 +269,8 @@ py.pkgs.toPythonApplication (py.pkgs.buildAzureCliPackage {
   meta = with lib; {
     homepage = "https://github.com/Azure/azure-cli";
     description = "Next generation multi-platform command line experience for Azure";
+    changelog = "https://github.com/MicrosoftDocs/azure-docs-cli/commit/a80f1a316226cf7801a25b7adb497ac13f875281";
```

Comment:
```suggestion
    changelog = "https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/release-notes-azure-cli.md";
```

Reply:
Is it sure that we want to put the main branch for changelog, it will get updated automatically but release notes won't always be in sync with the version.

I thought pinning the exact commit was better, and then I'd update it at every version bump so it always stays coherent.

Reply:
You didn't refer to the filename and referring to the latest changelog avoids having to change it and users can find the versions themselves.